packages/i18n/locales/cs/translation.json

```diff
@@ -0,0 +1,629 @@
+      "noSubDaosYet": "Zatím neexistuje žádné SubDAO",
+      "noVote": "Ne",
+      "none": "Nic",
+      "notAdmin": "Vaše DAO není správcem této smlouvy. V takovém případě nebude návrh proveditelný.",
+      "notMet": "Nesplněno",
+      "nothingFound": "Nic nebylo nalezeno.",
+      "numNftsSelected_one": "{{count}} vybrané NFT",
+      "numNftsSelected_other": "{{count}} vybraných NFTs",
+      "numProposals_one": "{{count}} návrh",
+      "numProposals_other": "{{count}} návrhů",
+      "numSelected_one": "{{count}} vybrán",
+      "numSelected_other": "{{count}} vybráno",
+      "openProposalsInInbox_one": "Máte {{count}} neodhlasovaný návrh.",
+      "openProposalsInInbox_other": "Máte {{count}} neodhlasovaných návrhů.",
+      "passing": "Prochází",
+      "pending": "Čeká se na",
+      "pendingRewards": "Očekávané odměny",
+      "percentVotingPower": "{{percent}} síly hlasů",
+      "preReleaseExplanation": "Jedná se o předběžnou verzi DAO DAO běžící na {{chain}} .",
+      "productVersion": "DAO DAO v{{versionNumber}}",
+      "proposalAllAbstainClarification": "Pokud se všichni hlasující zdrží hlasování, návrh neprojde.",
+      "proposalDepositWillBeRefunded": "Záloha za návrh bude vrácena po jeho uzavření.",
+      "proposalQuorumTooltip": "Aby byl návrh schválen, musí pro něj hlasovat tento podíl hlasovacích práv.",
+      "proposalStatus": {
+        "notOpen": "O tomto návrhu již nelze hlasovat. {{turnoutPercent}} všech odevzdaných hlasů a {{turnoutYesPercent}} z nich bylo ano..{{extra}}",
+        "willFail": "Pokud současné hlasování proběhne, tento návrh nebude přijat.",
+        "willFailBadQuorum": "Pokud současné hlasování proběhne, tento návrh nebude přijat z důvodu nedostatečné účasti voličů.",
+        "willFailBadThreshold": "Pokud bude současné hlasování platit, tento návrh nebude přijat, protože nebyl odevzdán dostatečný počet hlasů pro.",
+        "willPass": "Pokud současné hlasování projde, bude tento návrh schválen."
+      },
+      "proposalThresholdTooltip": "Aby návrh prošel, musí získat tento podíl hlasů pro.",
+      "reached": "Dosaženo",
+      "recognizeSubDaoDescription": "Rozpoznávání {{name}} jako SubDAO se zobrazí na kartě SubDAOs.",
+      "refund": "Vrácení peněz",
+      "removeCw20FromTreasuryActionDescription": "Přestat zobrazovat zůstatek DAO tokenu CW20 v pokladně.",
+      "removeCw721FromTreasuryActionDescription": "Přestat zobrazovat NFT vlastněné DAO ze sbírky NFT CW721 v pokladně.",
+      "reviewYourProposal": "Prohlédněte si návrh...",
+      "reviewYourTransaction": "Prohlédněte si transakci...",
+      "searchActionPlaceholder": "Najít akci...",
+      "searchDraftPlaceholder": "Najít návrh...",
+      "searchNftsPlaceholder": "Najít NFT...",
+      "spendActionDescription_dao": "Utraťte tokeny z pokladny",
+      "spendActionDescription_wallet": "Utratit nativní tokeny z peněženky",
+      "stakeActionDescription": "Upravit staking uzamčení nativního tokenu.",
```

Comment:
chápu to tak, že staking se tady překládá jako uzamčení. tady je to najednou staking. asi by to chtělo upravit